### ISSUE TITLE
Support for individuals

### ISSUE THREAD

**Issue by dosumis** [geneontology/obographs#7]
Is it legal in obographs to include individuals in the 'existential graph'?  Something like this could work:

Type statement to named class:
"subj" : "Individual A",
"pred" : "InstanceOf",
"obj" : "Class B"

Type statement, simple anonymous class (R some B):
"subj" : "Individual A",
"pred" : "part_of",
"obj" : "Class B"

FACT:
"subj" : "Individual A",
"pred" : "part_of",
"obj" : "Individual B"

Simple reification pattern (A R Value B)
"subj" : "Class A",
"pred" : "has_exemplar",
"obj" : "Individual B"

**Comment by cmungall** [comment]
Yes, OPAs to edges is the most obvious, and is the undocumented behavior now

Not sure I follow the reification

CAs - own axiom type or an edge? I am tentatively opting for the latter

@kltm this differs slightly from ABoxes in MinervaJSON

**Comment by dosumis** [comment]
> Not sure I follow the reification

Seems clear enough to me:

ClassA has_exemplar value IndividualB
<-> 
"subj" : "Class A",
"pred" : "has_exemplar",
"obj" : "Individual B"

Can we try to keep to a single vocab for this?

OPA = FACT
CA (class assertion) = type axiom ?

**Comment by dosumis** [comment]
Would be fantastically useful for VFB if generation could be extended to cover at least the I:C and I:I examples above.

**Comment by cmungall** [comment]
```
Type statement, simple anonymous class (R some B):
"subj" : "Individual A",
"pred" : "part_of",
"obj" : "Class B"
```

This is a nice convenience pattern. However, the reverse mapping becomes ambiguous if we have punning. But this may not be a problem if we say punning out of scope: #11

**Comment by cmungall** [comment]
hasValue pattern understood (it was the reificiation terminology that threw me). Same comments as above.

**Comment by cmungall** [comment]
How should we handle negative property assertion axioms?
